File: HtmlEditableContent/Interop/src/markuprangeelement.ts
import { MarkUpRange } from "./markuprange";

export class MarkUpRangeElement extends MarkUpRange {

    constructor(positionStart: number, positionEnd: number, anchorStart: number, focusStart: number, id: string, tagName: string, textContent: string) {
        super(positionStart, positionEnd, anchorStart, focusStart, id);
        this.tagName = tagName;
        this.textContent = textContent;
    }
    public tagName: string
    public textContent: string
}

File: HtmlBuilder/MarkUpRangeElement.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace HtmlBuilder
{
    public class MarkUpRangeElement : MarkUpRange
    {
        public MarkUpRangeElement() { }

        public string TagName { get; set; }
        public string TextContent { get; set; }
    }
}
